server/views/pages/search/probationSearch/probationSearch.njk

```diff
@@ -19,7 +19,7 @@
         {% else %}
             {% set link = offender | createOffenderLink  %}
         {% endif %}
-        {% if offender.comName %}
+        {% if offender.comName or offender.kind == "TIME_SERVED" or offender.isOnProbation %}
```

Comment:
Why do we need offender.isOnProbation as this would never reach here in that case it would go to the /licence/vary/id/ on line 18

Reply:
That's for determining which link to use 
This check is for ensuring whether that the link is actually included